BenefitsAllocationUpload/App_Start/BundleConfig.cs

```diff
@@ -23,6 +23,8 @@ public static void RegisterBundles(BundleCollection bundles)
             bundles.Add(new ScriptBundle("~/bundles/modernizr").Include(
                         "~/Scripts/modernizr-*"));
 
+            bundles.Add(new ScriptBundle("~/bundles/bootstrap").Include("~/Content/bootstrap.min.css"));
```

Comment:
@taylorkj ~/bundles/ are for javascript, you want to use "~/Content/bootstrap" instead